workshop/library/ui/controllers/topbarInterface.lua

```diff
@@ -33,7 +33,11 @@ return {
             backgroundColour = globals.defaultColours.white,
         })
 
+<<<<<<< HEAD
```

Comment:
bad merge here too?

Reply:
Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.